### ISSUE TITLE
Remaining download time (ETA)

### ISSUE THREAD

**Issue by Shankymahajan** [feross/webtorrent-desktop#691]
You guys can try to put this feature . Very small feature but very useful feature btw I'm in love with this app

**Comment by mathiasvr** [comment]
Are you talking about showing the remaining time a video is playing for, or remaining download time (ETA)?

**Comment by pureocean** [comment]
Deat **WebTorrent-Desktop** Developers,

Fİrstly, Thank you to @**mathiasvr** and all who have contributed in this application. Better and better  It's wonderful.

Secondly, I'm sorry my bad English.

I think, all Torrent-Stream Players displays current and total time of media when it playing the audio and video. Except WebTorrent. Can be unobserved or ignored displays when current playing and Total Time (of audio or video)? I hope not. 

I look optimist. WebTorrrent-Desktop still be developed; many features step by step processing, adding. I hope, this feature implement sooner or later. 

Personally, I couldn't not turn changelog of every new build. I waiting in hope and my gut tells me: "_Perhaps, In the present instance, can be finally added to WebTorrent-Desktop displays of current playing & total time show feature_." 

The Current and Total time is very important while playing the audio & video.
For example format: 00:01:28 / 02:45:33

First one: (This segment shows the media's current duration while playing time on audio or video) 
Second one: (Total duraction of the audio or video. All right, no counting)

*

I have a request: **Jump to Time**

I often use 'the _Jump to Time/Frame_' box of **PotPlayer.** In this dalog-box , the user input here for jump, any specified time (hh:mm:ss OR only minutes:second, for example: _114:58_ ).. So the player jumps the media (audio or video) to the duration as manually inputed of the user.

As a matter of course, I am aware of adding **Time Jump** feature what its shouldn't be exigency in your implementation priority list.

**Comment by mathiasvr** [comment]
Playback progress time was added by @demoneaux in #562 and released as v0.6.0 (does not appear in changelog for some reason).
I moved your Time Jump request to a separate issue #692.

**Comment by Shankymahajan** [comment]
I mean to say for ETA . (remaining time to download a file)

**Comment by pureocean** [comment]
@mathiasvr, Thank you very much for your reply and info. I didn't know that added time and duration feature. I am so glad. Thank you and @demoneaux 

I also thank you for open as new issue my "Time Jump" feature request. I transfered from #691 to #692 my "**Time Jump**" request texts. I hope, I didn't make a mistake.

Therefore, the request link was changed. If you wish update, new the link: https://github.com/feross/webtorrent-desktop/issues/692#issuecomment-229766097

@mathiasvr,  thank you for all your help and interest.
